msteams-platform/tabs/how-to/authentication/tab-sso-overview.md

```diff
@@ -6,42 +6,42 @@ ms.localizationpriority: high
 ---
 # Enable SSO for tab app
 
-With SSO in Teams, app users have the advantage of using Teams to access tab apps. After logging into Teams using Microsoft or Microsoft 365 account, app users can use your app without needing to sign in again. Your app is available to app users on any device with access granted through Azure AD.
+With single sign-on (SSO) in Teams, the app users have the advantage of using Teams to access tab apps. After logging in to Teams using Microsoft or Microsoft 365 account, the app users can use your app without the need to sign in again. Your app is available to the app users on any device with the access granted through Azure Active Directory (Azure AD).
```

Comment:
[Grammar] This is a case where the definite article isn't appropriate, because we're talking about app users in general, not a specific set of them. The previous version of this paragraph, without the added "the", reads more naturally.

Reply:
Retained the previous version of this paragraph that reads more naturally.